### ISSUE TITLE
Release-type input overrides config file

### ISSUE THREAD

**Issue by kerruba-milkman** [google-github-actions/release-please-action#975]
### TL;DR

Having both `config-file` and `release-type` as inputs will make the `release-type` input override (or add) the release-type to all packages in the config file

### Detailed design

Currently when passing both the `release-type` and `config-file` inputs, the `config-file` input is actually ignored and a default configuration file is generated using the release-type input.
A scenario that I'm working on right know is to centralize the configuration file for different repositories - I always need to update certain 'extra-files' -  and override the release-type using a Variable in the repository. I thought this was possible, but found out reading the code is not the how the action is meant to be used.

If having the `release-type` + `config-file` merging strategy not  possible for any reason, at least documenting it in the README would be useful to avoid confusion

**Comment by collindutter** [comment]
@kerruba-milkman you're a life saver! I was tearing my hair out for hours trying to figure out why my config file was being ignored. Removing `release-type` from the action input fixed it.

**Comment by conventoangelo** [comment]
> Removing release-type from the action input fixed it.

Thank you @kerruba-milkman @collindutter. Had to test around 4 release-please PRs trying to find out why `config` is being ignored. This solved it for me!

**Comment by arunvisvajeetrs** [comment]
I have another question,
- If I don't mention the `release-type` in the github action config, then it expects the `.release-please-manifest.json` file and  `release-please-config.json`
- Since now I have to have the `.release-please-manifest.json`, I have to mention the version as { ".", "x.x.x" } even though it is not a monorepo
- And because of this, I have to have the config in the below format
````
{
  "packages": {
	".": {
		"release-type": "node",
		"include-v-in-tag": false
	}
}
```

- This causes the component name to be included in the tag 😖. Now I need to have another config 
`"include-component-in-tag": false` as well.

Is there not an option to make the config work without having to configure like a monorepo? I'd really like an answer or a fix for this? 

I found this issue very late (my bad) had to do a lot of trial and errors to reach here.
